crypto/x509/x_x509.c

```diff
@@ -101,6 +101,13 @@ static int x509_cb(int operation, ASN1_VALUE **pval, const ASN1_ITEM *it,
         ASN1_OCTET_STRING_free(ret->distinguishing_id);
         break;
 
+    case ASN1_OP_DUP_POST:
+        {
+            X509 *old = exarg;
+
+            ret->libctx = old->libctx;
+            ret->propq = old->propq;
```

Comment:
Drive by comment: I note that X509_new_with_libctx() doesn't "dup" the propq, and hence you don't here either. We probably should.

Reply:
Its on my list..

Reply:
I will do this once this PR is merged..

Reply:
Reported as #13486 so we do not forget.